src/Test/PdbUtilities/Writer/CustomMetadataSymUnmanagedWriter.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DiaSymReader;
+
+namespace Roslyn.Test.PdbUtilities;
+
+internal sealed class CustomMetadataSymUnmanagedWriter(SymUnmanagedWriter target, byte[] customMetadata) : DelegatingSymUnmanagedWriter(target)
```

Comment:
>CustomMetadataSymUnmanagedWriter [](http://example.com/codeflow?start=22&length=32)

nit: if we don't feel this type will be used more generally, consider moving into `PDBTests.cs`

Reply:
I think it needs to be in this separate utility assembly, because in PDBTests some of the types would conflict between Microsoft.DiaSymReader and Microsoft.CodeAnalysis which are both referenced from there